### PR TITLE
USHIFT-1879: Test harness - change PS4 to improve debuggability 

### PR DESCRIPTION
**Pull request by pmtk**

### CODE REVIEW

File: test/bin/ci_phase_iso_boot.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # This script runs on the hypervisor to boot all scenario VMs.
 
 set -xeuo pipefail
-PS4='+ $(date "+%T.%N")\011 '
+export PS4='+ $(date "+%T.%N") $BASH_SOURCE:$LINENO \011'
```
Comment (pacevedom) [inline test/bin/ci_phase_iso_boot.sh:6]: What about removing $HOME from $BASH_SOURCE?

Comment (pmtk) [reply test/bin/ci_phase_iso_boot.sh]: Done

**Review by pacevedom — APPROVED** [review]
/lgtm

**Pull request merged**